stores/src/main/java/com/issoft/training/stores/utils/StorePopulator.java

```diff
@@ -0,0 +1,24 @@
+package com.issoft.training.stores.utils;
+
+import com.issoft.training.domain.shop.categories.Category;
+import dataBase.DataBaseStorePopulator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StorePopulator {
+
+    public List<Category> storePopulator(String populator) {
+        List<PopulatorMethod> populatorClass = new ArrayList<>();
```

Comment:
it is better to create map with populators where key is a enum. and create this map in constuctor/static block, not in the method body